boards/common-uboot.mk

```diff
@@ -33,6 +33,14 @@ define uboot_bootdir
     MTOOLSRC=$1 mcopy $5 $2boot/dtb
 endef
 
+# Create an empty configuration directory in the boot partition
+#
+# $1 is the mtools config file
+# $2 is the mtools drive letter
+define uboot_confdir
+    MTOOLSRC=$1 mmd $2conf.d
```

Comment:
If you just add this line to the uboot_bootdir macro above, then you dont have to edit every single board config to make it happen.  That macro is already described as the place that all needed boot partition actions are done.

Reply:
I wanted to make `conf.d` creation as an explicit step because it is not required as part of boot. I know the alternative works, but this was done intentionally to not mix two things, even though they all end up in the same boot partition.

Reply:
In that case, create a super macro ("uboot_doitall"?) that calls both uboot_bootdir and uboot_confdir - they both take the same params and reducing the amount of lines in each board file makes it easier for others to see and port the important parts.